tests/src/performance/linkbench/getcert.cmd

```diff
@@ -0,0 +1,2 @@
+@echo off
+"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\amd64\dumpbin.exe" /headers %1 | findstr /C:"Certificates Directory
```

Comment:
Yes, I assume that you can get VS install location from the environment rather than assuming VS is installed on drive c:.